application.py

```diff
@@ -12,6 +12,5 @@ def register():
     password = request.form.get("password")    
     return render_template("register.html", name=name, password=password)
```

Comment:
how are you indicating registration is successful or not

Reply:
When anyone registers in the index.html and press register button, then it redirects to register.html where users can see that their registration is successful.  If they again try to register, it will show "This name already exists".